nestkernel/archiving_node.cpp

```diff
@@ -81,7 +88,9 @@ Archiving_Node::register_stdp_connection( double t_first_read )
   // For details see bug #218. MH 08-04-22
 
   for ( std::deque< histentry >::iterator runner = history_.begin();
-        runner != history_.end() && runner->t_ <= t_first_read;
+        runner != history_.end()
+          && ( t_first_read - runner->t_ > -1.0
+                 * kernel().connection_manager.get_stdp_eps() );
```

Comment:
Could you try to get the -1.0 on the same line as `kernel()....`? I think that would be good for legibility.

Reply:
Again, not possible due to coding style guidelines.